src/database/database-sqlite3.cpp

```diff
@@ -798,9 +798,9 @@ void ModMetadataDatabaseSQLite3::createDatabase()
 
 	SQLOK(sqlite3_exec(m_database,
 		"CREATE TABLE IF NOT EXISTS `pairs` (\n"
-			"	`modname` TEXT,\n"
-			"	`key` BLOB,\n"
-			"	`value` BLOB,\n"
+			"	`modname` TEXT NOT NULL,\n"
+			"	`key` BLOB NOT NULL,\n"
```

Comment:
Isn't this redundant due to the primary key constraint below?

Reply:
Apparently not: https://sqlite.org/lang_createtable.html#the_primary_key